Atomic/syscontainers.py

```diff
@@ -847,7 +846,6 @@ def extract_system_container(self, img, destination):
         repo = self._get_ostree_repo()
         if not repo:
             return False
-        self.args.remote = None
         return self._checkout_system_container(repo, img, img, 0, False, destination=destination, extract_only=True)
```

Comment:
sorry I placed the comment on the wrong line (how can I comment on a line that is not part of the patch?), the change must be done in the `update_system_container` function.  At line 448 we read the value of self.arg.remote which now is not honored in the call to `_checkout_system_container` at line 467.

The change remote=self.args.remote should go at the line 448